src/pages/index.tsx

```diff
@@ -88,6 +89,7 @@ const Home = ({ seoMeta }: PageProps) => {
       />
       <Testimonials />
       <WeAlreadyTaughtAt />
+      <FeedbackPopup type="GENERAL" />
```

Comment:
We don't a Feedback until use do actions. For General Feedbacks, We can plan a small component